weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsCharts.tsx

```diff
@@ -0,0 +1,133 @@
+import {GridFilterModel, GridSortModel} from '@mui/x-data-grid-pro';
+import React, {useMemo, useState} from 'react';
+
+import {
+  IconChevronDown,
+  IconChevronNext,
+  IconLightbulbInfo,
+} from '../../../../../Icon';
+import {Tailwind} from '../../../../../Tailwind';
+import {WFHighLevelCallFilter} from './callsTableFilter';
+import {useCallsForQueryCharts} from './callsTableQuery';
+import {
+  ErrorPlotlyChart,
+  LatencyPlotlyChart,
+  RequestsPlotlyChart,
+} from './Charts';
+
+type CallsChartsProps = {
+  entity: string;
+  project: string;
+  filterModelProp: GridFilterModel;
+  filter: WFHighLevelCallFilter;
+};
+
+export const CallsCharts = ({
+  entity,
+  project,
+  filter,
+  filterModelProp,
+}: CallsChartsProps) => {
+  const columns = useMemo(
+    () => ['started_at', 'ended_at', 'exception', 'id'],
+    []
+  );
+  const columnSet = useMemo(() => new Set(columns), [columns]);
+  const sortCalls: GridSortModel = useMemo(
+    () => [{field: 'started_at', sort: 'desc'}],
+    []
+  );
+  const calls = useCallsForQueryCharts(
+    entity,
+    project,
+    filter,
+    filterModelProp,
+    0,
+    1000,
+    columns,
+    columnSet,
+    sortCalls
+  );
+  const [isInsightsOpen, setIsInsightsOpen] = useState(false);
+
+  const toggleInsights = () => {
+    setIsInsightsOpen(!isInsightsOpen);
+  };
+
+  const chartData = useMemo(() => {
+    if (calls.loading || !calls.result || calls.result.length === 0) {
+      return {latency: [], errors: [], requests: []};
+    }
+
+    const data: {
+      latency: Array<{started_at: string; latency: number}>;
+      errors: Array<{started_at: string; isError: boolean}>;
+      requests: Array<{started_at: string}>;
+    } = {
+      latency: [],
+      errors: [],
+      requests: [],
+    };
+
+    calls.result.forEach(call => {
+      const started_at = call.traceCall?.started_at;
+      if (!started_at) {
+        return;
+      }
+      const ended_at = call.traceCall?.ended_at;
+
+      const isError =
+        call.traceCall?.exception !== null &&
+        call.traceCall?.exception !== undefined &&
+        call.traceCall?.exception !== '';
```

Comment:
Filtering out non-errors (so chart will start at the time the first error occurred)